.gitignore

```diff
@@ -2,3 +2,4 @@ npm-debug.log
 /node_modules
 /example/installed
 /example/data
+.idea/
```

Comment:
This seemingly has nothing to do with setting concurrency to 1, so it shouldn't be in this PR right?